decidim-core/lib/decidim/query_extensions.rb

```diff
@@ -15,7 +15,7 @@ def self.extend!(type)
           type !types[ProcessType]
           description "Lists all processes."
 
-          resolve ->(_obj, _args, ctx) {
+          resolve lambda { |_obj, _args, ctx|
```

Comment:
Well this style using lambdas is actually used everywhere else where `resolve` is used, so the change seems fine!